android/build.gradle

```diff
@@ -67,4 +104,37 @@ dependencies {
 	playImplementation 'com.google.android.gms:play-services-auth:20.7.0'
 }
 
+def ndkPath = project.hasProperty('ndkPath') ? project.property('ndkPath') : System.getenv('ANDROID_SDK_ROOT')
```

Comment:
I put up a more portable variant of this in https://github.com/tailscale/tailscale-android/tree/raggi/buildport